scripts/test-harness-contracts.ps1

```
@@ -235,6 +235,24 @@ try {
     Check-Match '.claude/skills/pelizzai-interview/SKILL.md' 'exactly one question per turn' 'interview-me asks exactly one question per turn'
     Check-Match '.claude/skills/pelizzai-interview/SKILL.md' 'Recommended:' 'interview-me highlights the best option'
 
+    # -- Issue #64: question delivery — prefer the native selector, recommendation in the label --
+    Check-Match '.claude/skills/pelizzai-interview/SKILL.md' 'native option-selection tool, use it' 'interview: prefers the platform native selector'
+    Check-Match '.claude/skills/pelizzai-interview/SKILL.md' 'where the user reads the choice' 'interview: recommendation lands where the choice is read'
+    Check-Match '.claude/skills/pelizzai-interview/SKILL.md' 'recommended option comes \*\*first\*\*[\s\S]{0,60}mark \*\*in its label\*\*' 'interview: recommended option is first AND marked in the label'
+    Check-Match '.claude/skills/pelizzai-discovery/SKILL.md' 'recommended option first, marked in its label' 'discovery: recommended option is first AND marked in the label'
+    Check-Match '.claude/skills/pelizzai-router/SKILL.md' 'accept first,\s+marked in its label' 'router: accept option is first AND marked in the label'
+    Check-Match '.claude/skills/pelizzai-discovery/SKILL.md' 'deliver the question through it' 'discovery: native delivery is preferred, not tolerated'
```

PSScriptAnalyzer (1.25.0)

[info] Cmdlet 'Check-Match' has positional parameter. Please use named parameters instead of positional parameters when calling a command.

(PSAvoidUsingPositionalParameters)